### ISSUE TITLE
Instruction to download jupyterlab-lsp on jupyterlab GUI is wrong

### ISSUE THREAD

**Issue by marimeireles** [krassowski/jupyterlab-lsp#568]
## What I am trying to do...

Install `jupyter-lsp`.

## How the project might keep the docs accurate...

If you're in jupyterlab and click on the GUI button to try to install the plugin you'll be greeted with the following text:

```
Server Companion

This package has indicated that it needs a corresponding server extension. Please contact your Administrator to update the server with one of the following commands:

pip install jupyterlab_lsp

conda install -c conda-forge jupyterlab_lsp

You should make sure that the indicated packages are installed before trying to use the extension. Do you want to continue with the extension installation?
```

This info is wrong: `conda install -c conda-forge jupyterlab_lsp`

```
 $ conda install -c conda-forge jupyterlab_lsp
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - jupyterlab_lsp

Current channels:

  - https://conda.anaconda.org/conda-forge/linux-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

Should be `jupyter-lsp`.

I don't know where to change this, but if you can point me out to it I can make this change.
Cheers :)
Thanks again for the awesome work!

**Comment by krassowski** [comment]
Thank you! Indeed, I overlooked it when merging https://github.com/krassowski/jupyterlab-lsp/pull/454. We need to edit:

https://github.com/krassowski/jupyterlab-lsp/blob/3a0236895b24530eb5f58c7ff932897bc2b9e157/packages/jupyterlab-lsp/package.json#L134

I commented on #454 suggesting `jupyter-lsp` (as it would be sufficient), but `jupyterlab-lsp` would work too (and might be actually better when it comes to updating in the future) IMO the issue comes from underscore.

**Comment by krassowski** [comment]
I think [python_packages/jupyterlab_lsp/jupyterlab_lsp/install.json](https://github.com/krassowski/jupyterlab-lsp/blob/master/python_packages/jupyterlab_lsp/jupyterlab_lsp/install.json) also needs changing to replace `_` with `-`.

**Comment by marimeireles** [comment]
>IMO the issue comes from underscore.

Yeah, I'd say so too.

**Comment by krassowski** [comment]
Closing as it was fixed by #570 and is now available in 3.6.0. Thanks @marimeireles!
